### PR TITLE
ZK-3932: zul.sel.Listbox.setSelInView() tries to access non-existing …

### PR DESCRIPTION
**Pull request by charlesqqq**
…DOM element

### CODE REVIEW

**Comment by charlesqqq** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

File: zktest/test/java/org/zkoss/zktest/zats/test2/B85_ZK_3932Test.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import org.zkoss.zktest.zats.WebDriverTestCase;
+
+public class B85_ZK_3932Test extends WebDriverTestCase {
+	@Test
+	public void test() {
+		connect();
+		click(jq(".z-button"));
+		waitResponse();
+		Assert.assertTrue(!jq(".z-error").exists());
```
Comment (scribetw) [inline zktest/test/java/org/zkoss/zktest/zats/test2/B85_ZK_3932Test.java]: There is an `assertFalse` method

Comment (charlesqqq) [reply zktest/test/java/org/zkoss/zktest/zats/test2/B85_ZK_3932Test.java]: ok

**Comment by klyvechen** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**